server/src/test/java/io/crate/integrationtests/CorrelatedSubqueryITest.java

```diff
@@ -152,9 +152,15 @@ public void test_correlated_subquery_within_case_using_outer_column_in_where_cla
                 END AS base_type
             FROM
                 pg_catalog.pg_type t
+            LIMIT 4
             """;
         execute(stmt);
-        assertThat(TestingHelpers.printedTable(response.rows())).isEqualTo("");
+        assertThat(TestingHelpers.printedTable(response.rows())).isEqualTo(
```

Comment:
This assertion was wrong and the collector wasn't implemented correctly. Re-using `FirstColumnConsumers` fixed this.